.php_cs

```diff
@@ -0,0 +1,59 @@
+<?php
+$header = <<<'EOF'
+
+ This file is part of the `liip/LiipImagineBundle` project.
+
+ (c) https://github.com/liip/LiipImagineBundle/graphs/contributors
+
+ For the full copyright and license information, please view the LICENSE.md
+ file that was distributed with this source code.
+EOF;
+
+$config = PhpCsFixer\Config::create()
+    ->setRiskyAllowed(true)
+    ->setRules([
+        //'@PHP56Migration' => true,
+        '@Symfony' => true,
+        //'@Symfony:risky' => true,
```

Comment:
What does this enable? Do we possibly want to do so? We generally strictly follow Symfony's coding standings, short of long array syntax and a few other small differences.

Reply:
It is commented, just in case that you or someone want enable to experiment. The risky enable more agressive fixes, that not in all cases could be perfect. According to doc "Risky rule is a rule, which could change code behaviour. By default no risky rules are run."